packages/react-components/src/Flag.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017-2021 @polkadot/app-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { Tag } from '@polkadot/react-components/index';
+import { FlagColor } from '@polkadot/react-components/types';
+
+interface FlagProps {
+  className?: string;
+  color: FlagColor;
+  label: React.ReactNode;
+}
+
+function Flag ({ className = '', color, label }: FlagProps): React.ReactElement<FlagProps> {
+  return (
+    <Tag
+      className={`${className} ${color === 'theme' ? ' highlight--color-bg highlight--bg' : ''}` }
+      color={color}
+      isFlag
+      label={label}
+      size='tiny'
+    />
+  );
+}
+
+export default React.memo(styled(Flag)`
+  border-radius: 0 0.25rem 0.25rem 0;
+  padding: 0.5833em 1.25em 0.5833em 1.5em;
+  font-size: 0.78571429rem;
```

Comment:
This sizing should really be applied to tags as well. Same height, same font. (Have not checked the font now, but at least the height s is ms different - not for this PR)